internal/controller/postgrescluster/pgmonitor.go

```diff
@@ -399,11 +428,7 @@ func configureExporterTLS(cluster *v1beta1.PostgresCluster, template *corev1.Pod
 		}}
 
 		exporterContainer.VolumeMounts = append(exporterContainer.VolumeMounts, mounts...)
-		exporterContainer.Env = append(exporterContainer.Env, corev1.EnvVar{
-			// TODO (jmckulk): define path not dir
-			Name:  "WEB_CONFIG_DIR",
-			Value: "web-config/",
-		})
+		exporterContainer.Command = append(exporterContainer.Command, "--web.config.file=/web-config/web-config.yml")
```

Comment:
It doesn't look like this flag is added to the expected command in the tests. I'm also not seeing this flag added when I deploy locally.

Is it being used? Why not define it above with the other flags?

Reply:
It only gets added when custom tls is used.

Reply:
It would be nice to have the command defined in the same place :pick:

Reply:
I'm not sure how that would work... :thinking:

Reply:
Oh sorry, I didn't realize this was a different function :sweat_smile: 

Where it is now makes sense.